src/reynir_correct/errtokenizer.py

```diff
@@ -104,6 +104,27 @@
     "Desember",
 )
 
+ACRONYMS = (    # HÍ og HA ganga kannski ekki hér
+    "Dv",
+    "Rúv",
+    "Byko",
+    "Íbv",
+    "Pga",
+    "Em",
+    "Ví",
+    "Mr",
+    "Mh",
+    "Ms"
+    "Hr",
```

Comment:
What about "Hr. Hnetusmjör hélt tónleika í Kópavogi í kvöld"?

Reply:
The period prevents 'Hr.' from being corrected and the same goes for the two other sentences, 'Mr.' and 'Ms.'. I have added testing for these sentences to show how they are handled.